operator/pkg/controllers/master/kubecontrollermanager.go

```diff
@@ -96,6 +93,10 @@ func kcmPodSpecFor(controlPlane *v1alpha1.ControlPlane) v1.PodSpec {
 					v1.ResourceCPU: resource.MustParse("1"),
 				},
 			},
+			Ports: []v1.ContainerPort{{
+				ContainerPort: 10252,
```

Comment:
I thought the insecure port has been deprecated in 1.19? what happen if we choose to use `KubernetesVersion=v1.21`?